server.js

```diff
@@ -0,0 +1,9 @@
+import app from './app.js';
```

Comment:
you've inverted control. Actually you have to run app.js file and server should start inside it. But you run server.js file to start the app. correct me if I'm wrong.
This way you are loosing a possibility to easily replace server.js file and the way you are starting server
And what will you do if you need to set up some logging or put a event handling on server or shut it down depending on anything? where will you put the logic?